style.css

```diff
@@ -0,0 +1,136 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;1,700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+/* Reset */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 65.5%;
+  display: flex;
+  flex-direction: column;
+}
+
+body {
+  line-height: 1;
+  overflow-x: hidden;
+}
+
+.padding-left-right {
+  padding: 0 2.4rem;
+}
+
+.header {
+  display: flex;
+  grid-template-columns: 1fr auto;
+  align-items: center;
+  height: 4.8rem;
+  background-color: #3c3a39;
+  color: #fff;
+}
```

Comment:
- [ ] Please I highly recommend sticking the required [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=HQA60gDOdof1M29t-0). 😄 

_**REQUIRED FIGMA DESIGN**_

![Screenshot (243)](https://user-images.githubusercontent.com/78104544/235163288-d07a8f32-6678-48cf-86fa-fb228c93ddfe.png)


_**YOUR DESIGN**_

- [ ] Please I suggest adding `justify-content: space-between` property to the `header` class of your `style.css` file to fit the required figma design. 😄 

![Screenshot (242)](https://user-images.githubusercontent.com/78104544/235163412-aebeb484-04de-41e8-b466-4bce731842f0.png)